### PR TITLE
[12.0] event_session: Migration to 12.0

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to create sessions associated with events.

Since the dependency to event_mail was embedded in this module, we've migrated it as well as the core features of 12.0 doesn't fully cover all the cases.

- [x] event_mail https://github.com/OCA/event/pull/145

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis and runbot

**Comment by chienandalu** [comment]
:green_apple: CI fixed

**Review by victor-ditay — APPROVED** [review]
LGTM

File: event_session/wizards/wizard_event_session.py [hunk 1]
```diff
@@ -181,6 +182,7 @@ def action_generate_sessions(self):
 
 class WizardEventSessionHours(models.TransientModel):
     _name = "wizard.event.session.hours"
+    _description = "Hours in wich the sessions will run"
```
Comment (pedrobaeza) [inline event_session/wizards/wizard_event_session.py:185]: s/wich/which

**Review by pedrobaeza — APPROVED** [review]
No migration scripts are needed in v11?

**Comment by chienandalu** [comment]
Yes, @pedrobaeza , and I'm detecting that the event_mail dependendy was merged inside the module instead of using the new core method for the same purpuse. I put this on hold since that needs to be done.

**Comment by pedrobaeza** [comment]
Well, I have seen that there's an inheritance of `event.mail`, so I thought all that code was needed.

**Comment by chienandalu** [comment]
Yeah, that misled me as well, but this https://github.com/odoo/odoo/blob/12.0/addons/event/models/event_mail.py#L26
must substitute this:
https://github.com/OCA/event/blob/11.0/event_session/models/event_mail.py#L95
using event_types to provide the default email schedulers.

**Comment by pedrobaeza** [comment]
OK, all yours!

**Comment by pedrobaeza** [comment]
@chienandalu, should we merge this or the mail question is still opened?

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by chienandalu** [comment]
Done and :green_heart:

**Review by emagdalenaC2i — APPROVED** [review]
Tested in runbot

**Comment by pedrobaeza** [comment]
@chienandalu I insist: no migration script is needed?

**Comment by chienandalu** [comment]
> @chienandalu I insist: no migration script is needed?

It shouldn't, i think.

**Comment by pedrobaeza** [comment]
OK, it's for having confirmation as no answer was given.

**Comment by chienandalu** [comment]
True, sorry :)

**Pull request merged**